### DISCUSSION TITLE
Cannot reuse one binding in the same SQL Query

### DISCUSSION THREAD

**Discussion by kovrom** [Budibase/budibase#4768] [Help]
Self hosted. 
Updated from  1.0.26 to 1.0.73. Every PostgreSQL query where one binding was used multiple times stopped working. Work around for now is to create multiple bindings with the same values. But it looks like a bug and definitely not very productive. Any info on that? Thank you.
For example following will not work anymore. Let's say `binding = 0`:
```
 SELECT something AS ValueOne, 
(SELECT something2 FROM table WHERE column ='{{binding}}') AS ValueTwo
FROM table
WHERE column2 ='{{binding}}';
```
To make it work I have to create 2 bindings: `binding=0` and `binding1=0`. And do:
```
 SELECT something AS ValueOne, 
(SELECT something2 FROM table WHERE column ='{{binding}}') AS ValueTwo
FROM table
WHERE column2 ='{{binding1}}';
```

**Comment by mjashanks** [comment]
Thanks @kovrom - I have reproduced this. Will log and get this fixed.

**Reply by kovrom** [reply]
Thank you

**Reply by mjashanks** [reply]
logged here: https://github.com/Budibase/budibase/issues/4779

**Reply by kovrom** [reply]
It was also logged here https://github.com/Budibase/budibase/issues/4767

**Reply by mjashanks** [reply]
Ah - thank you for that :)
